modules/ztp-acm-adding-images-to-mirror-registry.adoc

```diff
@@ -36,7 +36,7 @@ $ export ROOTFS_IMAGE_NAME=<rootfs_image_name> <2>
 +
 [source,terminal]
 ----
-$ export OCP_VERSION=<ocp_version> <2>
+$ export OCP_VERSION=<ocp_version> <3>
```

Comment:
I didn't even know that you could do callouts like this, across multiple code blocks and have them all together at the end :thinking: